c10/cuda/CUDAFunctions.cpp

```diff
@@ -16,7 +16,7 @@ int32_t driver_version() {
   return driver_version;
 }
 
-int device_count_impl() {
+int device_count_impl(bool check_no_driver) {
```

Comment:
It's a little hard to tell what `check_no_driver` is supposed to mean. `fail_if_no_driver` would be clearer, at least to me